kong/plugins/datadog/handler.lua

```diff
@@ -25,59 +25,52 @@ local get_consumer_id = {
   end,
   username    = function(consumer)
     return consumer and consumer.username
-  end
+  end,
 }
 
 
 local metrics = {
-  status_count = function (api_name, message, metric_config, logger)
-    local fmt = string_format("%s.request.status", api_name,
-                       message.response.status)
+  status_count = function(name_prefix, message, metric_config, logger, tags)
+    local fmt = string_format("%srequest.status", name_prefix, message.response.status)
 
     logger:send_statsd(string_format("%s.%s", fmt, message.response.status),
-                       1, logger.stat_types.counter,
-                       metric_config.sample_rate, metric_config.tags)
+                       1, logger.stat_types.counter, metric_config.sample_rate, tags)
```

Comment:
Please follow the code style.
https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#code-style

Please take care of this all over the patch.

Also, avoid unnecessary changes to keep the patch readable.